pkg/tfvars/baremetal/baremetal.go

```diff
@@ -75,13 +82,15 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 			"power_interface":      accessDetails.PowerInterface(),
 			"raid_interface":       accessDetails.RAIDInterface(),
 			"vendor_interface":     accessDetails.VendorInterface(),
+			"instance_info": map[string]interface{}{
```

Comment:
This is in the wrong spot, you'll see instanceInfo calculated later in this file. It's due to terraforms really limited data structures.

Reply:
That's the source of the current failures

Reply:
Ah! fixed